docs/src/pages/layout/grid/grid.md

```diff
@@ -61,7 +61,7 @@ Unfortunately, CSS grid is only supported by the most recent browsers.
 There is one limitation with the negative margin we use to implement the spacing between items.
 A horizontal scroll will appear if a negative margin goes beyond the `<body>`.
 There are 3 available workarounds:
-1. Not using the spacing feature and implementing it in user space `spacing={0}`.
+1. Not using the spacing feature and implementing it in user space `spacing={0}` (default).
```

Comment:
What does "user space" here mean? If I don't use spacing feature, why will I still need to implement it as `spacing={0}`? Does this sentence mean:

Not using the spacing feature **OR** implementing it in user space `spacing={0}` (default).